lib/rubocop/cop/rspec/capybara/feature_methods.rb

```diff
@@ -45,13 +45,15 @@ class FeatureMethods < Cop
             feature:    :describe
           }.freeze
 
-          def_node_matcher :feature_method?, <<-PATTERN
-            (send {(const nil :RSpec) nil} ${:#{MAP.keys.join(' :')}} ...)
+          def_node_matcher :feature_method, <<-PATTERN
+            (block
+              $(send {(const nil :RSpec) nil} ${:#{MAP.keys.join(' :')}} ...)
```

Comment:
I’m tempted to replace `:#{MAP.keys.join(' :')}` with `MAP.keys.map(&:inspect).join(' ')`, just out of personal taste. Having the (symbol prefix) colon as part of the `join` argument just rubs me the wrong way.

Reply:
@bquorning I think that's more elegant than my original implementation. 👍